<launch>

	<!-- Command line arguments -->
	<arg name="robot_id" />
	<arg name="shared_dir" />

	<!-- Service/Topic/File Names -->
	<param name="shared_dir"				type="string"	value="$(arg shared_dir)" />
	<param name="goal_pos_file_format"		type="string"	value="%s/goal_pos_%03d" />
	<param name="current_pos_file_format"	type="string"	value="%s/current_pos_%03d" />

	<!-- Plan Execution -->
	<node pkg="box" name="supervisor"		type="supervisor.py" respawn="false" output="screen">
		<param name="robot_id"				type="int"		value="$(arg robot_id)" />
		<param name="robot_pos_topic"		type="string"	value="/amcl_pose" />
		<param name="move_base_topic"		type="string"	value="move_base" />
		<param name="map_frame_id"			type="string"	value="/map" />
		<param name="robot_pos_cov_limit"	type="double"	value="0.05"/>
		<param name="localize_rotate_vel"	type="double"	value="0.5"/>
		<param name="action_exec_timeout"	type="double"	value="120"/>
	</node>

	<!-- Disabling Turtlebot Safety Controller -->
	<node pkg="rostopic" type="rostopic" name="rostopic" args="pub /kobuki_safety_controller/disable std_msgs/Empty"/>

</launch>

